[revealjs] Mention `title-fancy/title-slide.html` as the partial starter to use for the title slide. (#1265)
quarto-dev/quarto-web: docs/presentations/revealjs/advanced.qmd

      - title-slide.html
```

> Quarto natively supports an improved title slide which improves handling of author and affiliations. While it is backward compatible with ‘author’ and ‘institute’, we now recommend that you use standard authors metadata, which will render properly. To provide your own partials for `title-slide.html`, you can start from the source code for this [fancier title slide template partial](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-fancy/title-slide.html). Customize this template as required, then save the results to `title-slide.html` alongside your presentation. 

Reviewer: ```suggestion
Quarto natively supports a title slide which improves the handling of authors and affiliations. While it is backward compatible with `author` and `institute`, we now recommend that you use standard [author metadata](/docs/authoring/front-matter.qmd#authors-and-affiliations). To provide your own partial for `title-slide.html`, you can start from the source code for this [fancier title slide template](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-fancy/title-slide.html). Customize this template as required, then save the results to `title-slide.html` alongside your presentation. 
```

> If you prefer the pandoc's default for revealjs (that can be opt-in without setting custom `template-partials` using `title-slide-style: pandoc`), here is the source code for this [simpler title slide template partial](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-slide.html). Though this one does not leverage our improved handling for [Authors & Affiliations](/docs/journals/authors.qmd), so we recommend using the fancy one as a starter. 

Reviewer: ```suggestion
If you prefer pandoc's default title slide you can opt-in using `title-slide-style: pandoc` in your YAML --- this uses this [simpler title slide template partial](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/pandoc/title-slide.html). However, this simpler partial does not leverage our improved handling for [Authors & Affiliations](/docs/journals/authors.qmd), so we recommend using the fancy one as a starter for custom title slides. 
```
